HandleNHNotificationCallActivity/handler.ts

```diff
@@ -49,6 +49,7 @@ const assertNever = (x: never): never => {
  * For each Notification Hub Message calls related Notification Hub service
  */
 export const getCallNHServiceActivityHandler = (
+  // tslint:disable-next-line: no-unused-variable
```

Comment:
I can argue that of you are declaring a variable which you are not using at the moment, os a symptom of premature optimization 😉 .

Reply:
Actually I don't rebember why this line, we are using telemetryClient.. 

Actually this is the activity we are dismissing after this third split.. do you think we should do it in this PR or make a new one after?

Reply:
Ahhhh I see, we're dismissing this activity so it became empty. It's ok, we will get rid of it in another iteration